tests/test_grid3d/grid_generator.py

```diff
@@ -2,7 +2,7 @@
 
 import xtgeo
 
-indecies = st.integers(min_value=4, max_value=8)
+indecies = st.integers(min_value=4, max_value=6)
```

Comment:
indices? Don't think indecies is a proper word

Reply:
Fixed the misspelling